cmd/snap-seccomp/main.go

```diff
@@ -20,8 +20,8 @@
 package main
 
 //#cgo CFLAGS: -D_FILE_OFFSET_BITS=64
-//#cgo pkg-config: --static --cflags libseccomp
-//#cgo LDFLAGS: -Wl,-Bstatic -lseccomp -Wl,-Bdynamic
+//#cgo pkg-config: libseccomp
+//#cgo LDFLAGS: -lseccomp
```

Comment:
Unfortunately this is not so easy. We absolutely need the static linking. We are exploring options but dynamic linking cannot be used here because it breaks re-exec on distributions with older libseccomp.

Reply:
Okay. Let me know once the path forward is clear, and I can update the Debian patch. Thanks!